### DISCUSSION TITLE
I need some permission for slient join and quit.

### DISCUSSION THREAD

**Discussion by iambestzige** [Zrips/CMI#5501] [Feature Requests]
Like cmi.slient.join and cmi.slient.quit for hide join message and quit message

**Comment by CloudeLecaw** [comment]
They allready exist. Take a look at the wiki or use these

cmi.messages.disablelogin – Disables login message
cmi.messages.disablequit – Disables logout message

They only work if u let cmi handle login and logout Meassage.
